README.md

```diff
@@ -80,28 +87,43 @@ cargo make install
 Alternatively, you can run `cargo install` directly as long as you set the required environment variables and features.
 
 ```sh
-export CFG_RELEASE=1.45.0-nightly
+export CFG_RELEASE=nightly
```

Comment:
Good call on changing this so it doesn't continue to drift. For rustfmt compilation purposes this just needs to be set to _something_, doesn't have to match the actual value.